peachjam/auth.py

```diff
@@ -6,16 +6,22 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.views import redirect_to_login
 
+from peachjam.models import pj_settings
+
 
 class SocialAccountAdapter(DefaultSocialAccountAdapter):
     """Adapter that permits logins only from specific domains."""
 
-    ALLOWED_DOMAINS = ["laws.africa", "africanlii.org"]
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.allowed_domains = [
+            s.strip() for s in pj_settings().allowed_login_domains.split(" ")
```

Comment:
If you use `.split()` you don't need to worry about weird whitespace or strip the results, and an empty string works as expected.

```python
>>> "  foo bar   baz boom  ".split(" ")
['', '', 'foo', 'bar', '', '', 'baz', 'boom', '', '']
>>> "  foo bar   baz boom  ".split()
['foo', 'bar', 'baz', 'boom']
>>> "".split()
[]
>>> "".split(" ")
['']
>>> " ".split(" ")
['', '']
>>> " ".split()
[]
```

Reply:
awesome. thanks